__tests__/api.test.js

```diff
@@ -53,3 +53,40 @@ describe("GET/api", () => {
       });
   });
 });
+describe("GET/api/articles/:article_id", () => {
+  it("status: 200, responds with the article with relevant id with the correct data type", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body.article).toBe("object");
+        expect(body.article).toHaveProperty("article_id", expect.any(Number));
+        expect(body.article).toHaveProperty("title", expect.any(String));
+        expect(body.article).toHaveProperty("topic", expect.any(String));
+        expect(body.article).toHaveProperty("author", expect.any(String));
+        expect(body.article).toHaveProperty("body", expect.any(String));
+        expect(body.article).toHaveProperty("created_at", expect.any(String));
+        expect(body.article).toHaveProperty("votes", expect.any(Number));
+        expect(body.article).toHaveProperty(
+          "article_img_url",
+          expect.any(String)
+        );
+      });
+  });
+  it("status 400, responds with a message when the id datatype is not valid", () => {
+    return request(app)
+      .get("/api/articles/banana")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid info");
+      });
+  });
+  it("status 404, responds with a message if id doesn't exist", () => {
+    return request(app)
+      .get("/api/articles/8500")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Id 8500 Not Found");
```

Comment:
I believe we discussed in a seminar that the error messages should be descriptive but not overly specific. This would be fine to be 'Not found' as a message.